### ISSUE TITLE
Make fails

### ISSUE THREAD

**Issue by andrew2net** [CalConnect/csd-subscription-upgrade#2]
When I try to run `make clean all` it fails with the error:
```
export GLOBIGNORE=sources/cc-51005.adoc; \
	cp sources/cc-51005.* documents; \
	unset GLOBIGNORE
FILENAME=sources/draft-ietf-calext-subscription-upgrade.adoc; \
	echo "Compiling locally..."; bundle exec metanorma $FILENAME
Compiling locally...
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 37:53
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 53:61
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 59:106
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 86:53
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 86:96
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 101:48
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 110:92
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 115:68
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 128:70
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 139:46
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 230:108
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 232:80
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 312:77
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 328:100
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 328:181
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 335:61
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 341:58
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 341:103
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 345:64
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 354:74
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 360:74
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 364:76
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 366:60
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 376:52
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 376:163
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 385:78
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 400:84
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 427:61
[metanorma-ietf] attribute "quote-title" not allowed here; expected attribute "target" @ 435:50
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 452:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 463:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 474:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 485:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 498:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 511:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 522:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 532:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 542:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 552:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 563:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 573:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 583:80
[metanorma-ietf] found attribute "quote-title", but no attributes allowed here @ 594:96
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 625:50
[metanorma-ietf] attribute "numbered" not allowed here; expected attribute "toc" @ 632:49
/Users/andrej/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/tempfile.rb:133:in `initialize': No such file or directory @ rb_sysopen - /var/folders/qf/c8zl30p57pq5p7n1l5znydl80000gn/T/sources/draft-ietf-calext-subscription-upgrade.txt20191018-27408-pfosgf (Errno::ENOENT)
	from /Users/andrej/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/tempfile.rb:133:in `open'
	from /Users/andrej/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/tempfile.rb:133:in `block in initialize'
	from /Users/andrej/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/tmpdir.rb:130:in `create'
	from /Users/andrej/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/tempfile.rb:131:in `initialize'
	from /Users/andrej/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/tempfile.rb:291:in `new'
	from /Users/andrej/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/tempfile.rb:291:in `open'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-ietf-1.0.6/lib/metanorma/ietf/processor.rb:71:in `output'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-0.3.15/lib/metanorma/compile.rb:192:in `block in process_extensions'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-0.3.15/lib/metanorma/compile.rb:183:in `each'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-0.3.15/lib/metanorma/compile.rb:183:in `process_extensions'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-0.3.15/lib/metanorma/compile.rb:20:in `compile'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-cli-1.2.7/lib/metanorma/cli/compiler.rb:41:in `compile_file'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-cli-1.2.7/lib/metanorma/cli/compiler.rb:29:in `compile'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-cli-1.2.7/lib/metanorma/cli/compiler.rb:33:in `compile'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-cli-1.2.7/lib/metanorma/cli/command.rb:34:in `compile'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-cli-1.2.7/lib/metanorma/cli.rb:60:in `start'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/gems/metanorma-cli-1.2.7/exe/metanorma:20:in `<top (required)>'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/bin/metanorma:23:in `load'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/bin/metanorma:23:in `<main>'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/bin/ruby_executable_hooks:15:in `eval'
	from /Users/andrej/.rvm/gems/ruby-2.4.0/bin/ruby_executable_hooks:15:in `<main>'
make: *** [sources/draft-ietf-calext-subscription-upgrade.xml] Error 1
```
Any thoughts?

**Comment by ronaldtse** [comment]
@andrew2net very strange -- I ran the same command and worked. Could you try again with the latest pushes?

**Comment by andrew2net** [comment]
@ronaldtse just tried the lastest and got the error again. Seems it tries to read a `draft-ietf-calext-subscription-upgrade.txt` temporary file but it doesn't exist.

**Comment by andrew2net** [comment]
@ronaldtse seems it's the issue related to `metanorma`. I tried to compile the document and got the same error:
```
bundle exec metanorma sources/draft-ietf-calext-subscription-upgrade.adoc
...
/Users/andrej/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/tempfile.rb:133:in `initialize': No such file or directory @ rb_sysopen - /var/folders/qf/c8zl30p57pq5p7n1l5znydl80000gn/T/sources/draft-ietf-calext-subscription-upgrade.txt20191018-29831-1tv0wda (Errno::ENOENT)
...
```

**Comment by ronaldtse** [comment]
It's finally fixed now in 1b4e80b.
